cluster-operator/src/test/java/io/strimzi/operator/cluster/ResourceUtils.java

```diff
@@ -545,16 +545,27 @@ public static KafkaBridge createKafkaBridgeCluster(String clusterCmNamespace, St
     /**
      * Generate empty Kafka MirrorMaker 2.0 ConfigMap
      */
-    public static KafkaMirrorMaker2 createEmptyKafkaMirrorMaker2Cluster(String clusterCmNamespace, String clusterCmName) {
-        return new KafkaMirrorMaker2Builder()
+    public static KafkaMirrorMaker2 createEmptyKafkaMirrorMaker2Cluster(String clusterCmNamespace, String clusterCmName, Integer replicas) {
+        KafkaMirrorMaker2Builder kafkaMirrorMaker2Builder = new KafkaMirrorMaker2Builder()
                 .withMetadata(new ObjectMetaBuilder()
                         .withName(clusterCmName)
                         .withNamespace(clusterCmNamespace)
                         .withLabels(TestUtils.map(Labels.KUBERNETES_DOMAIN + "part-of", "tests",
                                 "my-user-label", "cromulent"))
-                        .build())
-                .withNewSpec().endSpec()
-                .build();
+                        .build());
+
+        if (replicas != null) {
+            kafkaMirrorMaker2Builder
+                    .withNewSpec()
+                        .withReplicas(replicas)
+                    .endSpec();
```

Comment:
```suggestion
                    .editOrNewSpec()
                        .withReplicas(replicas)
                    .endSpec();
```
As Jakub suggests

Reply:
The difference being that withNewSpec overrides, whereas edit or new, edits a pre-existing spec if it exists, or creates a new one if it doesn't.